test/server/drain_manager_impl_test.cc

```diff
@@ -17,10 +17,13 @@ namespace Envoy {
 namespace Server {
 namespace {
 
-class DrainManagerImplTest : public testing::Test {
+constexpr int DrainTimeSeconds(600);
```

Comment:
nit: just use std::chrono::seconds here. Feel free to do this change in one of your other PRs.